pservice/pdo/pservice/scripts/PServiceCLI.py

```diff
@@ -78,6 +78,10 @@ def __init__(self, config, enclave) :
             'dataRequest' : self._datareq,
         }
 
+        logger.warn('start provisioning server')
+        logger.warn('verifying_key: %s', self.PSPK)
+        logger.warn('encryption_key: %s', self.EncryptionKey)
```

Comment:
i'm surprised to find an encryption key for PS enclave. Is this there purely for symmetry reason to eservice enclaves?  Does anybody use that key? Given that ps enclave is "hidden" from outside and is more for plausible deniability of PS itself, having the PS encrypting any input to enclave doesn't really seem to make a difference in that argument? In any case, what seems sensitive is the enclave response but this should be encrypted the contract enclave encryption key which is presumably not this key referenced here?

Reply:
changed.